### PR TITLE
Remove Node from the Docker Compose

### LINKED ISSUE #11: Create a Docker compose to set up the local development environment
**What**: A Docker-containerized local environment including the proper setup of the Postgres database and the environment variables.

### PR DESCRIPTION
**Pull request by benhuangbmj**
1. Responding to #16 , remove Node from the Docker Compose
2. In turn, create the template of .env to help configure the database connection
3. Update `README.md` with the instruction on `.env.template`. (Only [this part](https://github.com/flushingtech/Votte_Backend/blob/4eb2b57953579c16ebd8d8c6b21557970cf2a76f/README.md?plain=1#L23-L25) is substantially changed. The other changes are from the code formatter.

Close #11 , #15 , #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +20,9 @@ This is the backend for Votte, which will allow Flushing Tech members to vote on
 
 All data is stored in a PostgreSQL database, ensuring reliable data management and integrity.
 
-Spin it up today with `docker-compose up -d`
+Spin it up today with `docker compose up -d`
+
+Rename `.env.template` to `.env`. The `DATABASE_URL` is configured to connect to the Docker PostgreSQL database, while the remaining values are _mock placeholders_ to enable spinning up the Node server. If you want to test the authentication flow, you must provide your own values.
```
Comment (wilrnh) [inline README.md:25]: very helpful documentation, thanks!

you can also include a link to the google oauth instructions: https://support.google.com/cloud/answer/6158849?hl=en

File: .env.template [hunk 2]
```diff
@@ -0,0 +1,4 @@
+DATABASE_URL=postgres://postgres:mysecretpassword@localhost:5432/mydatabase?sslmode=disable
+GOOGLE_CLIENT_ID=google-client-id
+JWT_SECRET=jwt-secret
+SERVER_URL=server-url
```
Comment (wilrnh) [inline .env.template:4]: Always good to document public settings, such as environment variables, that the users will interact with 👍

```suggestion
# DB connection URL
DATABASE_URL=postgres://postgres:mysecretpassword@localhost:5432/mydatabase?sslmode=disable
# Google OAuth credentials
# 	See: https://support.google.com/cloud/answer/6158849?hl=en
GOOGLE_CLIENT_ID=google-client-id
GOOGLE_CLIENT_SECRET=google-client-secret
# JWT private key
# 	Generate using: `openssl rand -hex 32`
JWT_SECRET=jwt-secret
# API server URL
#	The external URL, usually used for the contruction of links
SERVER_URL=server-url
```

**Review by wilrnh — APPROVED** [review]
approved with some minor suggestions. great job!

File: README.md (hunk 1 above)
Comment (benhuangbmj) [reply README.md:25]: Got you. Link this to #22

File: .env.template (hunk 2 above)
Comment (benhuangbmj) [reply .env.template:4]: Got you. Link this to #23 .

**Pull request merged**